style.css

```diff
@@ -438,3 +439,188 @@ textarea {
   justify-content: left;
   cursor: pointer;
 }
+
+#navtag {
+  display: none;
+}
+
+/* Styling for desktop version of project */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    box-sizing: 0;
+  }
+
+  #wrapper {
+    width: 100%;
+    background-size: contain;
+    margin: 0;
+  }
+
+  #navbar {
+    display: none;
+  }
+
+  #navtag {
+    display: block;
+  }
+
+  #container {
+    display: flex;
+    column-gap: 950px;
+    width: 100%;
+    margin: 0;
+  }
+
+  .navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: right;
+    column-gap: 30px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .logo {
+    margin-left: 55px;
+  }
+
+  #name {
+    width: 700px;
+  }
+
+  #section {
+    margin-left: 260px;
+    margin-bottom: 255px;
+  }
+
+  #about {
+    width: 920px;
+  }
+
+  .content {
+    display: flex;
+    padding-left: -50px;
+    justify-content: left;
+    height: 496px;
+    margin-left: 142px;
+    margin-right: 142px;
+    column-gap: 46px;
+  }
```

Comment:
- [ ] I think it would be better to improve replicate[ figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=lm66x04u5euYTBoh-0) on element spacing of your contents. It would be better to include CSS padding for ```top, right and bottom``` to create space around the element, inside any defined border. :joy: 
- [ ] Kindly try to reduce the space between your text to match that of Figma design :+1: 

_Your may follow this [link ](https://www.geeksforgeeks.org/css-margins-padding/)for more info :+1: :+1:_ 

**Your design**
![image](https://user-images.githubusercontent.com/102757126/206628207-5f677823-28cf-4e23-a2a5-520ab7355209.png)
**Figma design**
![image](https://user-images.githubusercontent.com/102757126/206628394-ea191ca7-49b9-4c78-b74b-fd6db4fa26f1.png)

Reply:
- [ ] Kindly work on the previous code reviewer's suggestion, not the issue addressed yet.